### DISCUSSION TITLE
(SUPPORT) PI-VPN Config on Tomato (Netgear R8000). NOT WORKING

### DISCUSSION THREAD

**Discussion by JetFn1** [pivpn/pivpn#1384] [Support]
Pi VPN server is up and running great! Loading the .OVPN file on my PC and Android Openvpn connects right away..

Trying to set up a client on my router to connect to the Piserver is proving to be impossible!! 

I have set up a VPN I pay for before on my router and it worked great... But for some reason I CANNOT get it to work with PIVPN. I went through the the whole .OVPN file and checked all the boxes on my router. I DO NOT KNOW WHY IT WILL NOT CONNECT!!

So any help would be appreciated. I will upload the Router settings along with the OVPN file taking out any personal info. If someone could help that would be great. I searched all over and could not get an answer specifically for a PIVPN config.

I don't know if I need to add custom config. I don't know if I need to have server verify. I know I entered the Static Key, CA, and Certs correct. The only thing I did not add was the Private Key. I added my username and password. So please tell me what I am missing.  

Thanks to everyone in advance.

![VPN Config](https://user-images.githubusercontent.com/26935779/112247497-c8c80980-8c2a-11eb-9ede-a4a3de974d58.jpg)
0-8c2a-11eb-8993-07475a4d9d04.jpg)

![Tomato 1](https://user-images.githubusercontent.com/26935779/112247882-67ed0100-8c2b-11eb-8cd7-4680654b0b46.jpg)
![Tomato 2](https://user-images.githubusercontent.com/26935779/112247266-63741880-8c2a-11eb-8b61-7a8e1f396354.jpg)
![Tomato 3](https://user-images.githubusercontent.com/26935779/112247273-65d67280-8c2a-11eb-9a36-83e14f44da5a.jpg)

**Comment by TinCanTech** [comment]
> I have set up a VPN I pay for before on my router and it worked great... But for some reason I CANNOT get it to work with PIVPN.

Because that is not supported.

**Comment by JetFn1** [comment]
> 
> 
> > I have set up a VPN I pay for before on my router and it worked great... But for some reason I CANNOT get it to work with PIVPN.
> 
> Because that is not supported.

Only supports using the OpenVpn app and not on a router?
